data/channels/dirk-kelly.yml

```diff
@@ -0,0 +1,5 @@
+name: Dirk Kelly
+slug: dirk-kelly
```

Comment:
This is why we're going to need to import based on the slug, so it knows which channel data file to add/update.